docs/guidance/language.md

```diff
@@ -0,0 +1,71 @@
+# Publishing data in your own language
+
+You can publish the value of free-text fields, for example `title`, `description` and `parties/name`, in your own language. You ought to set `language` to the language used in free-text fields in your data.
+
+In order for your data to be interoperable and compatible with OC4IDS tools and methodologies, you cannot:
+
+* Translate codes from OC4IDS's codelists, for example the `type` field needs to be set to a code from the [ProjectType codelist](../reference/codelists.md#projecttype), like 'construction', which cannot be translated.
+* Translate field names (keys), for example, you cannot translate `title` to `ชื่อ`.
+
+To check whether a field can be published in your own language, check the [internationalization lookup table](#internationalization-lookup).
+
+## Examples
+
+The following JSON snippet is valid OC4IDS data. `title` can be published in Thai because it is a free-text field:
+
+```json
+{
+  "id": "1",
+  "title": "ตัวอย่างโครงการ",
+  "type": "construction"
+}
+```
+
+The following JSON snippet is not valid because "การก่อสร้าง" is not a valid code from the [ProjectType codelist](../reference/codelists.md#projecttype):
+
+```json
+{
+  "id": "1",
+  "title": "ตัวอย่างโครงการ",
+  "type": "การก่อสร้าง"
+}
+```
+
+The following JSON snippet is not valid because "ชื่อ" and "พิมพ์" are not valid field names in OC4IDS:
+
+```json
+{
+  "id": "1",
+  "ชื่อ": "ตัวอย่างโครงการ",
+  "พิมพ์": "construction"
+}
+```
+
+## Tabular data
+
+In order to ease access for non-English speakers, you can publish a spreadsheet or CSV file with field and code titles from an OC4IDS translation. Currently, OC4IDS is available in English and Spanish. If you would like to translate the schema to your own language, please [contact the OC4IDS Helpdesk](mailto:data@open-contracting.org).
+
+The following CSV except uses field and code titles from the Spanish translation of OC4IDS:
+
+| Identificador o Referencia | Título del Proyecto | Tipo de proyecto |
+| -------------------------- | ------------------- | ---------------- |
+| 1                          | proyecto de ejemplo | Construcción     |
```

Comment:
I'm not aware that Flatten Tool knows about codelists.

Reply:
It doesn't. In the original text, Flatten tool was only introduced after the table and only field titles were mentioned in context of Flatten Tool. My thinking was that a publisher could substitute codes for titles as part of their publication pipeline, either before using Flatten Tool or, if they are using CSV format, after. Do you think that's worth explaining?

> In order to ease access for non-English speakers, you can publish a spreadsheet or CSV file with field and code titles from an OC4IDS translation. Currently, OC4IDS is available in English and Spanish. If you would like to translate the schema to your own language, please [contact the OC4IDS Helpdesk](mailto:data@open-contracting.org).
>
> The following CSV except uses field and code titles from the Spanish translation of OC4IDS:
> 
> | Identificador o Referencia | Título del Proyecto | Tipo de proyecto |
> | -------------------------- | ------------------- | ---------------- |
> | 1                          | proyecto de ejemplo | Construcción     |
> 
> You can use [Flatten Tool](https://flatten-tool.readthedocs.io/en/latest/) to generate a spreadsheet or CSV file with translated field titles; for example, the following command converts the example OC4IDS JSON file to xlsx format using field titles from the Spanish schema:

Reply:
Hmm, yes, I thought that might be the case.

We start the page by saying that you can't translate field names or codes. In this section, we say that you can use field *titles* instead of names, and that titles are OK to translate.

If we say that codes are OK to translate, then – since that contradicts the earlier rule – we will need a little more content to explain why it's OK in spreadsheets/CSVs.

That said, I'm not sure (one way or the other) whether it's OK to translate codes. I think it's OK if there is a existing, reversible tool, e.g. like an OCDS Kit command that uses published translations of codelists.

In the absence of a tool, we're counting on publishers to reliably use the codelist translations.

Anyhow, this might be moot, because the only publishers we know who might translate codes aren't Spanish language.

I figure we can leave it for now and open an issue for consideration.

Reply:
> I figure we can leave it for now and open an issue for consideration.

I've created an issue: https://github.com/open-contracting/infrastructure/issues/366